### ISSUE TITLE
`canproco` - add manually corrected T2w SC segmentation

### ISSUE THREAD

**Issue by valosekj** [neuropoly/data-management#186]
I added the `derivatives` folder with manually corrected T2w SC segmentation to the `datasets/canproco`. The branch is named `jv/canproco-add_t2w_sc_manual`.

@mchen1110 and @sandrinebedard thank you for the manual corrections!

SC segmentation was corrected for 45 subjects. The QC (`qc_corr_20221130140244.zip`) is attached to this issue. The list of corrected subjects (`2022-11-28_qc_fail.yml`) is in the attached QC zip.

[qc_corr_20221130140244.zip](https://github.com/neuropoly/data-management/files/10126913/qc_corr_20221130140244.zip)

**Comment by sandrinebedard** [comment]
I checked and everything looks in place!

@mguaypaq can you check it out?

**Comment by valosekj** [comment]
> For the curious, the `bids-validator` errors include:
> 
> * 2828 files with bad names (e.g., `sub-cal056_ses-M0_STIR.json`).

Yeah, `bids-validator` probably does not officially support the `STIR`/`PSIR` images yet. I encountered that also previously when working with ivadomed. See [here](https://github.com/ivadomed/ivadomed/issues/1166#issue-1270959717).

> * There's an extra space character (before the tab character) after `participant_id` on the first line of `participants.tsv`.
> * Lots of empty cells in `participants.tsv` that should instead be marked `n/a` (several in the `sex` column, and it looks like the `researcher` column is entirely empty).
> 
> and also several warnings, of which maybe these ones are relevant?
> 
> * No `README` file.
> * Participants with explicit age 89 or higher (is this relevant here?). Here's the warning description:
>   > As per section 164.514(C) of "The De-identification Standard" under HIPAA guidelines, participants with age 89 or higher should be tagged as 89+. More information can be found at https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/#standard

Thank you for these points! I think they are not induced by my PR. We should probably open a new issue and list them there?

**Comment by mguaypaq** [comment]
> I think they are not induced by my PR. We should probably open a new issue and list them there?

Sure! And/or re-open this one and push more commits to your existing branch, if you want, whatever's easier.

**Comment by jcohenadad** [comment]
a README file is extremely important @valosekj pls add one thanks

**Comment by valosekj** [comment]
> a README file is extremely important @valosekj pls add one thanks

Sure! README added in the `76c5fcb9e870564a101bba5bbc4f7820ce28ab05` commit within the `jv/canproco_cleanup` branch. Also mentioned in https://github.com/neuropoly/data-management/issues/187
